sandbox/shantanu/postgres-db.tf

```
@@ -0,0 +1,21 @@
+
+resource "azurerm_postgresql_server" "test-pg-db" {
+  name                = "test-pg-psqlserver"
+  resource_group_name = azurerm_resource_group.sandbox.name
+  location            = azurerm_resource_group.sandbox.location
+
+  administrator_login          = "myadmin"
+  administrator_login_password = "myadm!n"
+
+  sku_name   = "GP_Gen5_4"
+  version    = "11"
+  storage_mb = 640000
```

5 GB should be enough to try
```suggestion
  storage_mb = 5120
```